libs/langchain/langchain/vectorstores/astradb.py

```diff
@@ -130,6 +131,9 @@ def __init__(
                 available in Astra DB. If left out, it will use Astra DB API's
                 defaults (i.e. "cosine" - but, for performance reasons,
                 "dot_product" is suggested if embeddings are normalized to one).
+            pre_delete_collection (Optional[bool]): whether to delete the collection 
+                before creating it. If False and the collection already exists, 
+                the collection will be used as is. 
```

Comment:
You may want to clear the whitespace at end of line in these three lines (and possibly move this param to the "advanced" section, but I have no strong opinion on this).